resources/lib/NineAnimeBrowser.py

```diff
@@ -21,12 +22,24 @@ class NineAnimeBrowser(BrowserBase.BrowserBase):
     _EPISODES_RE = \
     re.compile("<li>\s<a.+?data-id=\"(.+?)\" data-base=\"(\d+)\".+?data-title=\"(.+?)\".+?href=\"\/watch\/.+?\">.+?</li>",
                re.DOTALL)
-    _EPISODE_BOXES_RE = \
-    re.compile("<i\sclass=\"fa\sfa-server\"></i>\s(.+?)\s</label>\s<div.+?>(.+?)</div>",
-               re.DOTALL)
-
     _EPISODE_LINK_RE = re.compile("<li><div><a\shref=\"/([-\w\s\d]+?)/(\d+?)\"\sclass=\"anm_det_pop\"><strong>(.+?)</strong></a><i\sclass=\"anititle\">(.+?)</i>", re.DOTALL)
 
+    settings = xbmcaddon.Addon(id='plugin.video.9anime')
```

Comment:
hmm, let's start with handling the reset of the comments then we will tackle this last..